actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -115,6 +115,7 @@ def _set_rendered_content_type(format) # :nodoc:
     # Normalize args and options.
     def _normalize_render(*args, &block) # :nodoc:
       options = _normalize_args(*args, &block)
```

Comment:
It seems like `&block` is never called in `_normalize_args` 🤷 
https://github.com/rails/rails/blob/8e90517d41b711859f37ce55910579a8675b1c0c/actionpack/lib/abstract_controller/rendering.rb#L75-L87

Reply:
I think you're right, I noticed the same thing when writing this PR.

I avoided modifying it since it was existing code and I wanted to keep my changes relatively scoped, but happy to update it if folks feel it should be (I think we could not pass it to `_normalize_args` at all).

Reply:
I'm not sure...
Interestingly it does get used here, and added to the options:
https://github.com/rails/rails/blob/436207aa2011463e49feb6310fa144bb0a96bf0c/actionpack/lib/action_controller/metal/rendering.rb#L87-L91
Not sure where the :update option name comes from and where it's used. 🤔 
Maybe you could use that as well for this PR?